packages/core/src/dialects/cockroachdb/connection-manager.ts

```diff
@@ -0,0 +1,33 @@
+import type { Sequelize } from '../../sequelize.js';
+import type { Connection } from '../abstract/connection-manager';
+import { AbstractConnectionManager } from '../abstract/connection-manager';
+
+export interface CockroachConnection extends Connection {
+  __loadDialectModule(moduleName: string): unknown;
+}
+
+export class CockroachdbConnectionManager extends AbstractConnectionManager<CockroachConnection> {
+  constructor(dialect: any, sequilize: Sequelize) {
+    super(dialect, sequilize);
+
+    /**
+      * As pg-types says, PostgreSQL returns everything as string
+      * Corrects this issue: https://github.com/cockroachdb/sequelize-cockroachdb/issues/50
+      */
+
+    const __loadDialectModule = this._loadDialectModule;
+    // const __loadDialectModule = super._loadDialectModule;
+    this._loadDialectModule = (...args) => {
+      const pg: any = __loadDialectModule(...args);
+      pg.types.setTypeParser(20, (val: any) => {
```

Comment:
Same here you can't make changes that would impact other dialects if yours is running, as users can run multiple instances of Sequelize with different databases

oid 20 is _int8, which should always be returned as a string (for now), not sometimes a string and sometimes a number. We'll eventually change that to use js bigints, but until we do it should remain as a string